Source/ThirdParty/gmock/gmock.xcodeproj/project.pbxproj

```diff
@@ -57,9 +74,6 @@
 			dstPath = "$(PUBLIC_HEADERS_FOLDER_PATH)/internal";
 			dstSubfolderSpec = 16;
 			files = (
-				440034652AD8C1BC0081CD73 /* gmock-internal-utils.h in Copy Headers Internal */,
-				440034662AD8C1BF0081CD73 /* gmock-port.h in Copy Headers Internal */,
-				440034672AD8C1C10081CD73 /* gmock-pp.h in Copy Headers Internal */,
```

Comment:
Please go ahead and remove this whole copy files phase, since we're now handling everything through the main Headers phase.